internal/cli/atlas/search/create.go

```diff
@@ -93,6 +94,8 @@ func CreateBuilder() *cobra.Command {
 		Annotations: map[string]string{
 			"indexNameDesc": "Name of the index.",
 		},
+		Example: fmt.Sprintf(`  # Create a search index named myIndex for the cluster named myCluster using a JSON index configuration file named search-config.json:
+  %s clusters search indexes create myIndex --clusterName myCluster --file search-config.json --output json`, cli.ExampleAtlasEntryPoint()),
```

Comment:
@gssbzn Can we specify an index name when using a json file? If not I will remove "myIndex"